showcase/src/main/kotlin/com/recco/showcase/main/PaletteSelection.kt

```diff
@@ -99,22 +99,17 @@ fun PaletteSelection(
                 shadowElevation = 2.dp
             ) {
                 Column {
-                    MosaicSampleSection(
-                        ReccoPalette.Fresh,
-                        showIconHeader = true,
-                        selectionClickPalette = {
-                            expanded = false
-                            selectionClickPalette(it)
+                    listOf(ReccoPalette.Fresh, ReccoPalette.Ocean, ReccoPalette.Spring, ReccoPalette.Tech)
```

Comment:
If this keeps growing, it might be needed for the "in screen" current design to be rethinked

Reply:
I know, I know xD